src/auction.js

```diff
@@ -168,7 +168,7 @@ export function newAuction({adUnits, adUnitCodes, callback, cbTimeout, labels, a
       _auctionEnd = Date.now();
 
       events.emit(CONSTANTS.EVENTS.AUCTION_END, getProperties());
-      bidsBackCallback(_adUnitCodes, function () {
+      bidsBackCallback(_adUnits, function () {
```

Comment:
@omerBrowsi - Please provide some background on this core change.

Reply:
RTD module is the place this hook is used, so it won't affect anything else (the hook was made for this module)
The change sends as parameter the adUnits objects (that hold all data on the ad units in this auction) instead of just ad units codes.
Both variables already exist n auction.js, I just choose to send different one as a parameter